roxctl/netpol/connectivity/map/map.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 const (
-	defaultOutputFileNamePrefix = "connlist."
-	defaultOutputFormat         = "txt"
+	defaultOutputFileNamePrefix  = "connlist."
+	exposureOutputFileNamePrefix = "exposure."
```

Comment:
shouldn't this prefix `exposure.` be removed?

Reply:
had a comment about this previously, (marked resolved) now marked as unresolved..

Reply:
yes it should, the rebase was not successful (some commits were not picked)
i am working on fixing this